src/types/error/test.rs

```diff
@@ -2,58 +2,133 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
+use chrono::Utc;
+
 use super::*;
+use db::delivery_problems::{ProblemSubtype, ProblemType};
 
 #[test]
-fn bad_request() {
-    let error: AppError = AppErrorKind::BadRequest.into();
-    assert_eq!(error.code(), 400);
-    assert_eq!(error.error(), "Bad Request");
-    assert!(error.errno().is_none());
+fn internal() {
```

Comment:
Oh, one more small collateral improvement. Previously we didn't have test coverage in her for all our error types. Now we do.